transformer.py

```diff
@@ -116,26 +116,19 @@ class SimplePerfherderTransformer(Transformer):
     def transform(self, data):
         self.entry_number += 1
         return {
-            'data': [data['suites'][0]['value']],
-            'xaxis': self.entry_number
+            'data': {
+             'value': data['suites'][0]['value'],
+             'xaxis': self.entry_number
+            }
         }
 
     def merge(self, sde):
-        merged = {'data': [], 'xaxis': []}
+        merged = {'data': []}
+        data = [entry['data'] for entry in sde]
 
-        for entry in sde:
-            if type(entry['xaxis']) in (dict, list,):
-                raise Exception(
-                    "Expecting non-iterable data type in xaxis entry, found %s" % type(entry['xaxis'])
-                )
-
-        data = [(entry['xaxis'], entry['data']) for entry in sde]
-
-        dsorted = sorted(data, key=lambda t: t[0])
-
-        for xval, val in dsorted:
-            merged['data'].extend(val)
-            merged['xaxis'].append(xval)
+        dsorted = sorted(data, key=lambda t: t['xaxis'])
```

Comment:
I do not know if you must be responsible for sorting. Only the person performing the analysis will know how best to show the data

Reply:
We don't need the sorting anymore with the `data` field containing everything (I had this here to sort the data and xaxis fields).